modules/edge-governance/src/governance.rs

```diff
@@ -106,6 +109,9 @@ decl_module! {
 			let hash = T::Hashing::hash(&buf[..]);
 			ensure!(<ProposalOf<T>>::get(hash) == None, "Proposal already exists");
 
+			// Burn the proposal creation bond amount
+			ensure!(T::Currency::can_slash(&_sender, Self::proposal_creation_bond()), "Not enough currency for slashing bond");
```

Comment:
We should switch `can_slash` to `can_reserve` here, see: https://github.com/paritytech/substrate/blob/master/srml/support/src/traits.rs#L353

Reply:
Ah I can remove it entirely. Check now.